### PR TITLE
Adding Client Sign-in front-end webpage

### PR DESCRIPTION
**Pull request by hiichloee**
Have had a call with Arman, we agreed that I'll take on the Sign-in page, and he'll handle the Sign-up page.
We're doing the client sign-in/up pages seperately.

Notice: the current version of the Sign-in page doesn't have the backend connection implemented yet. The form is displayed correctly, but it's not connected to Django for any actual authentication or data handling. It's essentially just a front-end form for now. But it does work for navigation purposes, and the link to the next page is functional.

### CODE REVIEW

**Review by jbm1412 — CHANGES_REQUESTED** [review]
Screenshot (1985)

The sign-in interface has turned yellow due to Dark Mode. Please refer to the Team Sign-In Page and follow the colour template to ensure it matches the current website's style.

Additionally, I noticed that in Arman's PR (PR 213), the sign-up process requests a Deakin email, while your sign-in page asks for a personal email. Since this is a collaborative effort, please discuss this discrepancy and make the necessary changes.

**Comment by hiichloee** [comment]
I’ve made the required updates to the sign-in page as your feedback:

1.Changed the email input field to request a Deakin email instead of a personal email.
2.Updated the label from "Passport" to "Password."

However, regarding the Dark Mode adjustments to align with the Team Sign-In Page's color template, I’m currently unsure how to implement the necessary changes. I don’t have sufficient experience with Dark Mode styling at this stage. While I’m willing to learn and work on it later, I’m unable to address this part right now.

**Comment by hiichloee** [comment]
Hi team, do i need to send a new pull request for these two?

File: xss_attempts.log
Comment (jbm1412) [inline xss_attempts.log:1]: Please delete this file!

Comment (jbm1412) [inline xss_attempts.log:1]: Please delete this file as it is overwriting

**Review by jbm1412 — CHANGES_REQUESTED** [review]

**Review by jbm1412 — APPROVED** [review]
Screenshot (1993)

Looks Good!

**Review by s-bonnor — APPROVED** [review]
Looks good, will be merge in a little bit

**Pull request merged**